bubblewrap.c

```diff
@@ -1378,6 +1380,17 @@ parse_args_recurse (int          *argcp,
           argv += 1;
           argc -= 1;
         }
+      /* should be the last nominal parameter */
+      else if (strcmp (arg, "--exec-filename") == 0)
+        {
+          if (argc < 2)
+            die ("--exec-filename takes an argument");
+
+          opt_exec_filename = argv[1];
+	  argv++;
+	  argc--;
+	  break;
```

Comment:
This `break` can go away if it isn't required to be the last option used. I don't think it actually needs to be?